packages/browser/jest.config.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
   ...require("../../jest.config"),
   // Allows running tests from packages/browser/
-  testEnvironment: "<rootDir>/../../tests/environment/customEnvironment.js",
+  setupFilesAfterEnv: ['<rootDir>/../../jest.setup.js'],
```

Comment:
You shouldn't need this again as it's in the root config.

Reply:
`rootDir` doesn't resolve to the same directory whether the `jest` command runs from the root or from a sub-package of the monorepo. Overriding this here allows to run `npm run test` from `packages/browser/`, which is useful when you don't want to run the whole test suite. If this isn't overriden there, `setupFilesAfterEnv` points to `packages/browser/jest.setup.js`.

Reply:
it'd be better to set `rootDir` option, I think, and specify that as `rootDir: "../../"` I think?

Reply:
Doing so runs all the test suites, and not only the target one, so it doesn't quite solve the issue.

Reply:
Ah, alrighty!